pypesto/visualize/ensemble.py

```diff
@@ -413,3 +414,59 @@ def _create_patches(
         )
 
     return patches_both_hit, patches_lb_hit, patches_ub_hit, patches_none_hit
+
+
+def ensemble_parameters_plot(
+    ensemble: Ensemble,
+    ax: Optional[plt.Axes] = None,
+    size: Optional[tuple[float]] = (12, 6)
+):
+    """
+    Visualize parameter ensemble.
+
+    Parameters
+    ----------
+    ensemble:
+        ensemble of parameter vectors (from pypesto.ensemble)
+    ax:
+        Axes object to use.
+    size:
+        Figure size (width, height) in inches. Is only applied when no ax
+        object is specified.
+
+    Returns
+    -------
+    ax: matplotlib.Axes
+        The plot axes.
+    """
+
+    if ax is None:
+        fig, ax = plt.subplots(figsize=size)
+
+    x = -0.4
```

Comment:
What is x?

Reply:
the x coordinate for the rectangle vertex, changed now